core/src/main/web/app/styles/dashboard.scss

```diff
@@ -22,6 +22,11 @@
     margin-top: 40px;
     margin-bottom: 60px;
   }
+  padding-top: 76px;
```

Comment:
Actually I also just found out that the original number was 60, not 76. Must have gotten mixed up. Will change to 60.